Changelog.md

```diff
@@ -3,6 +3,9 @@
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](
 https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**10.4.0**
+- gitlab: upgrade CE to v10.4.0
```

Comment:
@solidnerd Please also include that you replaced base Docker image with `sameersbn/ubuntu:16.04.20180124` which upgrades to ubuntu 16.04 from 14.04